src/magic_folder/test/test_web.py

```diff
@@ -365,6 +366,7 @@ def magic_folder_config(author, local_directory):
         u"collective-dircap": u"URI:DIR2-RO:{}:{}".format(b2a("\0" * 16), b2a("\1" * 32)),
         u"upload-dircap": u"URI:DIR2:{}:{}".format(b2a("\2" * 16), b2a("\3" * 32)),
         u"poll-interval": 60,
+        u"scan-interval": None,
```

Comment:
Is `None` different from `null` and `0`?

Reply:
`null` is JS terminology and is basically the same as `None` (i.e. a `null` in some JSON will become a `None` on the python side and vice-versa). Python doesn't have "`null`".

Reply:
Thanks meejah!

My comment was about our using the three terms seemingly interchangeably.  I was a bit uncompromising maybe with having the word `null` used in docs meaning the number `0`. Users will probably figure that one out.

From my reading the patch I believe currently sometimes `None` and sometimes `0` means _do not scan at all_.
It should be one of those two; I personally like `0` here better since it's of the same type as the other valid values.

Reply:
Yeah, definitely "choose one" I'd say.
I also prefer `0` to mean "disable scanning" (which is also somewhat common for these sorts of options IME).

Reply:
I think the only place where `0` was mentioned was in the `--help` output (and it isn't supported. I've removed that mention of it. `0` shouldn't be supported, and `null` (in the API/db)/ `None` in python means no scanning.